app/src/main/java/ch/epfl/sdp/contamination/ConcretePositionAggregator.java

```diff
@@ -55,7 +56,11 @@ private void startTimer() {
         class UpdatePosTask extends TimerTask {
             public void run() {
                 if (isOnline && newestLocation != null) {
-                    registerPosition(newestLocation, newestDate);
+                    if(!IN_TEST){

```

Comment:
using the reverse condition in a if with two branches is not ideal for me but I won't ask you to re run the CI just for that

Reply:
Oh yes indeed